xsd/pmo/people.xsd

```diff
@@ -73,6 +73,18 @@ SOFTWARE.
           <xs:restriction base="xs:boolean"/>
         </xs:simpleType>
       </xs:element>
+      <xs:element name="skills">
+        <xs:complexType>
+          <xs:sequence>
+            <xs:element name="skill" minOccurs="0" maxOccurs="unbounded" type="xs:string"/>
```

Comment:
@g4s8 let's set `maxOccurs` to `4` in order to prevent this list from being too long